### PR TITLE
Add Order Reference and Callback documentation

### PR DESCRIPTION
**Pull request by florianebmeier**
- [ ] Merge branch `change-account-to-customer` first, then rebase and check links. #8

### CODE REVIEW

**Review by Magnus93 — APPROVED** [review]

File: docs/merchant/integration-guide/balance.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+### Adding funds to Balance
+Adding funds to the balance can be done via the regular **Order Endpoint ???**. 
+To charge the balance of an customer, specify the `charge` field of the Customer payment as `"balance"`.
+Once the payment is authorized, it will automatically credit the balance with the total of the payment and [charge](#) the order.
+
+An order that failed authorization will set the Custoomer status to `"pending"` and retry authorization as specified in [retrying failed customer orders](#).
+
+#### Request
+``` {1}
+POST https://merchant.intergiro.com/order
```
Comment (Magnus93) [inline docs/merchant/integration-guide/balance.md:38]: Change to: 
```
POST /v1/order

Host: merchant.intergiro.com
Type-Content: application/json
Authentication: ....
```

File: docs/merchant/integration-guide/balance.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+If the balance already partially paid for the items specified in the "balance" field of the customer, that amount will be added to the order as its own item.
+
+The order will be an customer order with the "charge" field in the "payment" field set to "balance" and will be automatically charged. 
+Only a successful payment will update the "total" field of the customer. 
+A failed payment will retry authorization as specified in [`retrying failed customer orders.`](../order#retry-failed-account-payments)
+
+#### Request
+
+```{1}
+DELETE https://merchant.intergiro.com/customer/<customerId>/balance
```
Comment (Magnus93) [inline docs/merchant/integration-guide/balance.md:72]: Change here as-well
```
DELETE /v1/customer/<customerId>/balance

Host: merchant.intergiro.com

File: docs/merchant/integration-guide/change-contact-information.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Change Contact Information
+
+To change the contact information associated to an customer make a `PUT` request to the following endpoint. 
+A request to this endpoint can either be made with a `"private"` authorization key or with the `"customer"` authorization key. 
+With the `"customer"` authorization key you have to specify `"me"` as the customer id, for `"private"` authorization specify the customer id of the customer you wish to change.
+The body of the request should be a valid [`Contact`](../../integrate/acquiring/reference.html#contact) data type.
+#### Request
+```{1}
+PUT /customer/<customerId>/contact
```
Comment (Magnus93) [inline docs/merchant/integration-guide/change-contact-information.md:9]: Add `/v1/customer/...`, `Host` and `Content-Type`

And add this to all other API calls
(`Content-Type: application/json` is only neccesery when sending a JSON)

**Review by Magnus93 — CHANGES_REQUESTED** [review]

File: docs/merchant/reference/order.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
+# Order
+
+## Creatable
+| Property   | Type                                                                 | Description                                                | Optional |
+|------------|----------------------------------------------------------------------|------------------------------------------------------------|----------|
+| `number`   | `string`                                                             | Order number in your system.                               | Yes      |
+| `customer` | [`string | Contact`](../../integrate/acquiring/reference#contact)    | Customer contact information or [customer id](./customer). | Yes      |
+| `items`    | [`number | Item | Item[]`](../../integrate/acquiring/reference#item) |                                                            |          |
+| `currency` | [`Currency`](../../integrate/acquiring/reference#currency)           |                                                            |          |
```
Comment (marcushanses) [inline docs/merchant/reference/order.md:9]: Add category.

**Review by marcushanses — CHANGES_REQUESTED** [review]

File: docs/merchant/reference/callback.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Callback
+
+## Receiving callbacks
+
+When creating an [`order`](./reference/order) you can provide a callback url under the field [`callback`]().
```
Comment (marcushanses) [inline docs/merchant/reference/callback.md:5]: Fix the link.

**Review by marcushanses — CHANGES_REQUESTED** [review]

**Review by marcushanses — APPROVED** [review]

**Review by Magnus93 — APPROVED** [review]

**Comment by Magnus93** [comment]
Rebase
Also let README file function be called `mounted`, not `created`

**Pull request merged**